test/sqllogictest/chbench.slt

```diff
@@ -1129,6 +1130,7 @@ ORDER BY supplier_cnt DESC
 | Join %4 %9 %10 (= #17 #23 #24)
 | | implementation = Differential %9 %10.(#0) %4.(#17)
 | | demand = (#17, #20..#22)
+| Filter if isnull(#17) then {null} else {true}
```

Comment:
This is a bit weird, because `#17` was present in a join key, and is almost certainly constrained to be non-null as a result (not strictly speaking, because differential joins just match on equality, but based on where the join likely came from). We should probably track that down and see!